### PR TITLE
Fourth milestone

### PR DESCRIPTION
**Pull request by bren2102**
Kindly request a code review for the fourth milestone of this calculator project.

1. Logic folder with calculate.js and  operate.js created
2. calculate.js module implemented
- Contains one method: calculate
- Calculate method accepts data object and buttonName
- The data object contains three properties: total, next, and operation.
- This method uses the buttonName argument to mutate the calculator’s data object and return its modified version.
3. operate.js module implement
- Import big.js module
- Implement one function: operate
- The operate function accepts numberOne, numberTwo, and operation as arguments.
- Uses the value of the operation argument
- If the buttonName in calculate.js is equal to any operation (e.g. +, -, x, ÷, % [modifier - turning the number into a percentage based number]), operate.js is used to calculate the total property.

4. calculate.js imported to the App component

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    case 'x':
+      return (nOne.times(nTwo)).toString();
+    case '÷':
+      if (nOne !== 0 && nTwo !== 0) {
+        return (nOne.div(nTwo)).toString();
+      }
+      return '0';
+    case '%':
+      if (nOne !== 0 && nTwo !== 0) {
+        return ((nOne.div(100)).times(nTwo)).toString();
```
Comment (Oghenebrume50) [inline src/logic/operate.js]: I think you need to turn the number into a percentile here and the only way to do that is to divide by 100, no need multiplying with the second number. percentage works with just one input 
see

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const nTwo = Big(numberTwo);
+  switch (operation) {
+    case '+':
+      return (nOne.plus(nTwo)).toString();
+    case '-':
+      return (nOne.minus(nTwo)).toString();
+    case 'x':
+      return (nOne.times(nTwo)).toString();
+    case '÷':
+      if (nOne !== 0 && nTwo !== 0) {
```
Comment (Oghenebrume50) [inline src/logic/operate.js]: 0 / num is well known in mathematics as zero, the only thing you need to check is the other way around so num / 0 this is when we have a problem

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @bren2102 awesome job, this is well written, however, I have pointed out some few changes, please look into those and when you are done request for another review :slightly_smiling_face: 

Please make changes and **request for another review**


**RAPHAEL NORIODE** :mask: stay_safe

**Review by suretrust — APPROVED** [review]
Hi @bren2102, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future milestones and happy coding!_ 🎉

**Pull request merged**